zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -956,6 +956,18 @@ property, when available, is noted below.
     and restart ZooKeeper process so ZooKeeper can continue normal data
     consistency check during recovery process.
     Default value is false.
+* *audit.enabled* :
```

Comment:
There's a type here: the property name is `enable` without the "D".

Reply:
corrected it, thanks